pkg/wallet/transfer_output.go

```diff
@@ -0,0 +1,21 @@
+package wallet
+
+import (
+	"math/big"
+
+	"github.com/CityOfZion/neo-go/pkg/util"
+)
+
+// TransferOutput respresents the output of a transaction.
+type TransferOutput struct {
```

Comment:
The `transaction` package inside `core` has an `Output` type that represents the output of a transaction.